### PR TITLE
i2pd: add runit service and system user

### PR DESCRIPTION
**Pull request by byouki-onna**
this is how this is done, right?

### CODE REVIEW

**Comment by byouki-onna** [comment]
Done.

File: srcpkgs/i2pd/template [hunk 1]
```diff
@@ -11,6 +11,8 @@ maintainer="Obosob <obosob@riseup.net>"
 makedepends="zlib-devel boost-devel libressl-devel miniupnpc-devel"
 distfiles="https://github.com/PurpleI2P/i2pd/archive/${version}.tar.gz"
 checksum=4d7946e33a6a1dd4439590ea23b494f36e6fbcb81f7b36857bf264413a19e9db
+system_groups="_i2pd"
```
Comment (Vaelatern) [inline srcpkgs/i2pd/template]: Redundant, you get this by having a `system_account`

File: srcpkgs/i2pd/files/i2pd/run [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+if test ! -d /var/lib/i2pd
+	then
+	mkdir -p /var/lib/i2pd
+	chown -R  _i2pd:_i2pd /var/lib/i2pd
+fi
```
Comment (Vaelatern) [inline srcpkgs/i2pd/files/i2pd/run]:
```
if [ ! -d /var/lib/i2pd ]; then
    mkdir -p /var/lib/i2pd
    chown -R  _i2pd:_i2pd /var/lib/i2pd
fi
```
is a little cleaner.

**Comment by Vaelatern** [comment]
If you are experienced with git, please make this just one commit with the commit message:
`i2pd: add runit service and system user`
Otherwise, just say so and we can use github to do it at merge time.

**Comment by byouki-onna** [comment]
Im afraid this is my first time really using git aside from 'git clone'. Sorry.

**Comment by Gottox** [comment]
@Vaelatern @xtraeme PR ok?

**Pull request merged**